Makefile

```diff
@@ -20,7 +20,7 @@ feed                 :; ./scripts/check-oracle-feed.sh $(pip)
 feed-lp              :; ./scripts/check-oracle-feed-lp.sh $(pip)
 wards                :; ./scripts/wards.sh $(target)
 time                 :; ./scripts/time.py date="$(date)" stamp="$(stamp)"
-exec-hash            :; ./scripts/hash-exec-copy.sh "$(if $(date),$(date),$(shell date +'%Y-%m-%d'))"
+exec-hash            :; ./scripts/hash-exec-copy.py date="$(date)"
```

Comment:
Why remove the conditional here?

Reply:
I think this `if` logic should be handled inside the python code itself, to avoid using platform-specific `date` command. As the whole point of this PR is to make the check platform-agnostic